src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -475,7 +493,15 @@ public function getFlags()
             ];
         }
 
-        Hook::exec('actionProductFlagsModifier', [
+        if ($show_out_of_stock) {
```

Comment:
```suggestion
        if ($this->shouldShowOutOfStockLabel($this->settings, $this->product)) {
```

No need to create a variable, you use it only once and the name of the function is quite self-explanatory.

Reply:
Fixed

Reply:
I don't see the modification ?